lib/galaxy/config/__init__.py

```diff
@@ -132,6 +133,14 @@ def _process_renamed_options(self, kwargs):
                 if old in kwargs and new not in kwargs:
                     kwargs[new] = kwargs[old]
 
+    def is_set(self, key):
+        """Check if a configuration option has been explicitly set."""
+        # NOTE: This will check all supplied keyword arguments, including those not in the schema.
+        # To check only schema options, change the line below to `if property not in self._raw_config:`
+        if property not in self._kwargs:
+            log.warning("Configuration option does not exist: '%s'" % key)
```

Comment:
Definitely not for debugging. As for needed or not, we had a discussion with @mvdbeek a while ago: 
https://github.com/galaxyproject/galaxy/pull/8867#pullrequestreview-306394120
My preference is to err on the side of caution, and I'd even raise an exception (see discussion^). However, if it safe to log a warning or even ignore this, I don't mind. @natefoo - `config.is_set('foo')`replaces `config.foo_set` - should we not guard/guard/guard aggressively against a case where `foo` is misspelled by a dev?

Reply:
It could have made sense in #8867 to log this message when you were checking if the key is in `self._raw_config`, but now you are checking if it's in `self._kwargs` and it doesn't make sense to log if it's just not set by the admin.

Reply:
Another whoops. It should be checking `self._raw_config`, of course! Thank you. I've accidentally overwritten your previous fix (key > property), added it w/this fix.